pyiron/base/generic/parameters.py

```diff
@@ -528,11 +528,7 @@ def from_hdf(self, hdf, group_name=None):
             hdf (ProjectHDFio): HDF5 group object
             group_name (str): HDF5 subgroup name - optional
         """
-        if group_name:
-            with hdf.open(group_name) as hdf_group:
-                data = hdf_group[self.table_name]
-        else:
-            data = hdf[self.table_name]
+        data = hdf[group_name or self.table_name]
```

Comment:
Is this really the same? It should be `hdf[group_name + "/" + self.table_name]` or `hdf[self.table_name]`

Reply:
I'm a bit confused still how to uniformly use the group_name in to_hdf.  This (my) way is definitely broken for how it is used, but the old way doesn't work with `ProjectHDFio.to_object()`.  I hadn't had the time to think about it and it should be moved to pyiron_base anyway.